### PR TITLE
Simplify some parts of README, add Cloud Foundry instructions

### PR DESCRIPTION
**Pull request by konklone**
This removes some things in the README I think it's simpler to assume (like how to install Python, Postgres, manage virtualenvs), and adds a section about how to deploy the app, now, with Cloud Foundry.

It also dings the `.python-version` down from `3.4.1` to `3.4.0`, since we're locked to `3.4.0` for the production runtime. For anyone using `pyenv` in development, it just makes sense to use the same Python version as the production server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,10 +89,14 @@ psql -d foia -c "CREATE USER foia WITH PASSWORD '<<PASSWORD>>';"
 django-admin.py syncdb
 ```
 
-* Finally, launch the server locally:
+* Update the `DATABASE_URL` in `.env` with your Postgres connection string.
+
+### Running the server
```
Comment (khandelwal) [inline README.md]: I think the 'running the server' section should come after the 'loading the data' section.

**Comment by khandelwal** [comment]
I liked the aspect of the README including all the Python, Django installation stuff (if you're coming to that environment new) - but also realize that not every project should have detailed instructions on how to do that. 

That said, should we (18F) put out a clean and useful set of instructions for deploying a Python/Django environment?

File: README.md (hunk 1 above)
Comment (konklone) [inline README.md]: Hmm, while I see your point, I also think it's important to verify as early as possible that the harder work (dependencies and project setup) are done. Could we merge this as-is and update it in a future PR?

**Comment by konklone** [comment]
I updated this PR with 7d5b043 to touch up the Ruby/Sass section a bit.

**Comment by konklone** [comment]
> That said, should we (18F) put out a clean and useful set of instructions for deploying a Python/Django environment?

You feel we have something to add to the public discourse that hasn't been covered better elsewhere? Or did you mean aimed at 18F itself?

**Pull request merged**